tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Astro uses Typescript by default. We've opted out as much as possible but found it really useful to validate markdown frontmatter as mentioned elsewhere

https://docs.astro.build/en/guides/typescript/